index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="css.css">
+
+    <title>Document</title>
+</head>
+<body>
+    <menu>
+    <div class="panel"><a>Home</a><br>
+        <a>Blog</a><br>
+        <a>About Me</a><br>
+        <a>Tech Stack</a><br>
+        <a>Workspace</a><br>
+        <a>Portfolio</a><br>
+        <a>Contact</a><br>
+        <a>FAQs</a></div>
+    <div class="block">
+        <div class="blockl">
+            <div class="blockl1"></div>
+            <div class="blockl1">Latest Articles
+                <div class="miniBlock"></div>
+                <div class="button"></div>
+            </div>
+        </div>
+        <div class="blockr">
+            <div class="blockr1">
+            </div>
+            
+            <div class="blockr1">
+                <div class="miniBlock"></div>
+            </div>
+        </div>
+        <div class="aboutMe">
```

Comment:
for class name and other css selectors, name convention is snake_case, or cobol-case